internal/entities/properties/constants.go

```diff
@@ -0,0 +1,31 @@
+//
+// Copyright 2024 Stacklok, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package properties
+
+// Repository property keys
+const (
+	RepoPropertyName   = "name"
+	RepoPropertyOwner  = "owner"
+	RepoPropertyRepoID = "repo_id"
+	// TODO: is this GH-specific? If yes, it should be moved under a provider-specific property
```

Comment:
Let's add the GitHub specific constants under GitHub. This should only have "standard" ones. e.g. we can keep "is_private", "is_archived", "is_fork".

Reply:
done, there's a separate patch now with just constants for generic entity that tracks with your PR #4178 then a bunch of provider-agnostic repo properties and then the rest of the properties have been moved under the provider